RELEASE_NOTES.md

```diff
@@ -275,7 +276,19 @@ In the process to extract the old initiatives vote form to a base handler a new
 
 For more information about the definition of a signature workflow read the documentation of `Decidim::Initiatives::SignatureWorkflowManifest`.
 
-### 2.6. [[TITLE OF THE ACTION]]
+### 2.6. Permission rename in surveys module
+
+As we have changed the terminology surveys from "respond" to "response", we need to make sure that your already set permissions are still working.
```

Comment:
The original term for this action was `answer`. I think `respond` didn't arrive to be in a release. 

```suggestion
As we have changed the terminology surveys from "answer" to "respond", we need to make sure that your already set permissions are still working.
```